src/SprykerEco/Zed/Unzer/Business/Credentials/UnzerCredentialsCreator.php

```diff
@@ -59,13 +80,38 @@ public function createUnzerCredentials(UnzerCredentialsTransfer $unzerCredential
         });
     }
 
+    /**
+     * @param \Generated\Shared\Transfer\UnzerCredentialsTransfer $unzerCredentialsTransfer
+     *
+     * @return \Generated\Shared\Transfer\UnzerCredentialsResponseTransfer
+     */
+    public function createUnzerCredentialsAndSetUnzerNotificationUrl(UnzerCredentialsTransfer $unzerCredentialsTransfer): UnzerCredentialsResponseTransfer
+    {
+        return $this->getTransactionHandler()->handleTransaction(function () use ($unzerCredentialsTransfer) {
+            $unzerCredentialsResponseTransfer = $this->executeCreateUnzerCredentialsTransaction($unzerCredentialsTransfer);
```

Comment:
Please, move all this code to a separate method `executeCreateUnzerCredentialsAndSetUnzerNotificationUrlTransaction`

Reply:
Yes, At least the "and..." part of the method name should be changed in my opinion and optionally the URL part should go into a separate method.